kludges.js

```diff
@@ -375,6 +373,12 @@ function restoreMethod(obj, name) {
 }
 
 var signals;
+var swipeTrackers = [
+    Main?.overview?._swipeTracker,
+    Main?.overview?._overview?._controls?._workspacesDisplay?._swipeTracker,
+    Main?.wm?._swipeTracker,
+    Main?.wm?._workspaceAnimation?._swipeTracker
+].filter(t => t !== undefined);
```

Comment:
```suggestion
].filter(t => typeof t !== "undefined");
```
Technically someone can do `var undefined = "foo"`. Unlikely, but `typeof` will never break this way.

Reply:
Thanks @smichel17, have implemented these changes (see latest commit).